src/index.js

```diff
@@ -35,6 +36,7 @@ module.exports = {
     }
   },
   getWixModulesList: () => wixModulesNames,
+  eventHandlersService,
```

Comment:
this is a bit risky... we will soon start exposing public apis that we didn't intend to.

Reply:
How will you make it less risky?